client/src/components/App.jsx

```diff
@@ -42,6 +42,9 @@ function App() {
 
   return (
     <div id="global">
+      <div id="Head" onClick={trackClick}>
```

Comment:
- also not an "action now" but need to have a consensus on whether to use ids for each major component, or to always use classNames
- originally it was to use ids, but, I think right now there's a mix